src/Optimizer/Transformer/ServerSideRendering.php

```diff
@@ -800,26 +754,17 @@ private function adaptBlockingAttributes(Document $document, DOMElement $ampElem
                             break;
                         }
 
-                        $customCss            = array_merge(
-                            $customCss,
-                            $this->extractSizesAttributeCss($document, $ampElement, $attribute)
-                        );
+                        $customCss += $this->extractSizesAttributeCss($document, $ampElement, $attribute);
```

Comment:
Is merging arrays in this way safe? In my experience it fails to add items that have the same array keys:

```php
$x = [ 1, 2, 3 ];
$x += [ 4, 5, 6, 7 ];
print_r( $x ); 
```

```
Array
(
    [0] => 1
    [1] => 2
    [2] => 3
    [3] => 7
)
```

Reply:
I tried to avoid `array_merge()` for performance reasons, but you're right, this doesn't seem reliable.